u1-request-response/app.js

```diff
@@ -0,0 +1,83 @@
+// Import libraries
+const express = require('express')
+
+// Setup app
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+app.get('/lab', (req, res) => {
+    res.send(`
+    <html><body>
+    <li>1. <a href="/profiles/parinya">/profiles/parinya</a></li>
+    <li>2. <a href="/api/profiles/parinya">/api/profiles/parinya</a></li>
+    <li>3. <a href="/cat-simulator">/cat-simulator</a></li>
+    <li>4. <a href="/admin/grades">/admin/grades</a> AND <a href="/api/grades">/api/grades</a> AND <a href="/boredom">/boredom</a></li>
+  </body></html>
+  
+    `)
+  })
+
+app.get('/profiles/parinya', (req, res) => {
+    res.send(`
+    <html><body>
+      <h1>Parinya Krongpha</h1>
+      <ul>
+        <li>Nickname : Chinjung</li>
+        <li>Like : Cat , Advaenture movie</li>
+        </ul>
+    </body></html>
+    `)
+})
+
+app.get('/api/profiles/parinya', (req, res) => {
+    res.json({
+      name : "Parinya Krongpha",
+      nickname : "Chinjung",
+      like : "Cat , Advaenture movie"
```

Comment:
เราอาจพิจารณาใช้ `Array` เช่น `['Cat', 'Adventure movie']`
We might also consider using an `Array`, like this: `['Cat', 'Adventure movie']`